tests/Unit/PrestaShopBundle/Service/DataProvider/Admin/CategoriesProviderTest.php

```diff
@@ -0,0 +1,474 @@
+                            'refMenu' => '518',
+                            'modules' => [],
+                            'subMenu' => [],
+                        ],
+                        'Traffic & Marketplaces' => (object) [
+                            'tab' => 'checkout',
+                            'name' => 'Traffic & Marketplaces',
+                            'refMenu' => '488',
+                            'modules' => [],
+                            'subMenu' => [],
+                        ],
+                        'Customers' => (object) [
+                            'tab' => 'administration',
+                            'name' => 'Customers',
+                            'refMenu' => '475',
+                            'modules' => [],
+                            'subMenu' => [],
+                        ],
+                        'Facebook & Social Networks' => (object) [
+                            'tab' => 'advertising_marketing',
+                            'name' => 'Facebook & Social Networks',
+                            'refMenu' => '455',
+                            'modules' => [],
+                            'subMenu' => [],
+                        ],
+                        'Specialized Platforms' => (object) [
+                            'tab' => 'others',
+                            'name' => 'Specialized Platforms',
+                            'refMenu' => '469',
+                            'modules' => [],
+                            'subMenu' => [],
+                        ],
+                        'other' => (object) [
+                            'tab' => 'other',
+                            'name' => 'Other',
+                            'refMenu' => 'other',
+                            'modules' => [$noTabsNoCategories],
+                            'subMenu' => [],
+                        ],
+                    ],
+                ],
+            ],
+            $this->provider->getCategoriesMenu(
+                [
+                    $gamification,
+                    $cronjobs,
+                    $noTabsNoCategories,
+                ]
+            )
+        );
+    }
+
+    private function mockModule(string $moduleName, string $tab = null, string $categoryName = null)
+    {
+        $mock = $this->getMockBuilder(Module::class)
+              ->getMock();
+
+        if ($tab !== null) {
+            $mock->attributes->set('tab', $tab);
+        }
+
+        if ($categoryName !== null) {
+            $mock->attributes->set('categoryParentEnglishName', $categoryName);
+        }
+
+        return $mock;
+    }
+
+    public function testGetParentCategoryWithoutResult()
+    {
+        $this->assertEquals('Test', $this->provider->getParentCategory('Test'));
+    }
+
+    public function testGetParentCategory()
+    {
+        $this->assertEquals('Administration', $this->provider->getParentCategory('Registration & Ordering Process'));
+    }
```

Comment:
Maybe you can add a test related to `CATEGORY_THEME` (when it's present and when it's unset)